rocAL/rocAL_pybind/examples/rocAL_api_caffe2_reader.py

```diff
@@ -72,22 +72,22 @@ def main():
     with pipe:
         if _rocal_bbox:
             jpegs, labels, bboxes = fn.readers.caffe2(
-                path=image_path, bbox=_rocal_bbox, random_shuffle=True)
+                path=image_path, bbox=_rocal_bbox)
         else:
             jpegs, labels = fn.readers.caffe2(
-                path=image_path, bbox=_rocal_bbox, random_shuffle=True)
+                path=image_path, bbox=_rocal_bbox)
         images = fn.decoders.image(jpegs, output_type=types.RGB, path=image_path, random_shuffle=True)
-        images = fn.resize(images, resize_x=224, resize_y=224)
-        pipe.set_outputs(images)
+        images = fn.resize(images, resize_width=224, resize_height=224)
+        pipe.setOutputs(images)
     pipe.build()
-    data_loader = ROCALClassificationIterator(pipe , display=display, device=device)
+    data_loader = ROCALClassificationIterator(pipe , display=0, device=device)
```

Comment:
Why is display hardcoded ? Add this variable back for display

Reply:
This display is for the classification iterator in generic.py which has its own draw_patches function. In order to plot the images using the path user provides, this is done